runtime/type.js

```diff
@@ -79,6 +79,16 @@ class Type {
     return this.variable;
   }
 
+  // TODO: rename SetView to Collection
+  // Once everything's moved over to this, we can change the
+  // underlying representation
+  get isCollection() {
+    return this.isSetView;
```

Comment:
Also wondered about adding a deprecation note to the not-collection method names in some manner, but that would probably produce extreme spew, so instead am just mentioning here.

Reply:
I discussed a less verbose deprecation with Scott this week. Once we figure something out I'll start to deprecate the old names.